c7n/resources/asg.py

```diff
@@ -1735,3 +1735,88 @@ def process_config(self, client, config):
             if e.response['Error']['Code'] == 'ValidationError':
                 return
             raise
+
+
+@resources.register('scaling-policies')
+class ScalingPolicies(query.QueryResourceManager):
+
+    class resource_type(query.TypeInfo):
+        service = 'autoscaling'
+        arn_type = "scalingPolicy"
+        id = name = 'PolicyName'
+        date = 'CreatedTime'
+        enum_spec = (
+            'describe_policies', 'ScalingPolicies', None
+        )
+        filter_name = 'PolicyNames'
+        filter_type = 'list'
+        cfn_type = config_type = 'AWS::AutoScaling::ScalingPolicy'
+
+
+class PolicyInfo:
```

Comment:
its unclear what the reuse potential for this is, we optimize for removability, not unused abstractions. i would suggest just merging this class into the filter below, we can refactor it back out if there are additional usages.